helpers/useReviewSectionRows.tsx

```diff
@@ -400,6 +418,13 @@ const useReviewSectionRows = <
                     </div>
                   ))}
                 </div>
+                <div>
+                  <NextLink href={changeLink.href} as={changeLink.as}>
+                    <a className="govuk-link" href={changeLink.as.pathname}>
+                      Change
+                    </a>
+                  </NextLink>
```

Comment:
Ok, so having said to use `NextLink`, I think we actually want to use `Link` from `lbh-frontend-react`, which means constructing the target URL including query parameters to pass in as its `href`. `Link` is set up to do this work properly (the href here is missing the query parameter), with the correct styling via `_app.tsxL20-53`.